### ISSUE TITLE
looks like an amazing module, but could not get it to work with 3.3* 

### ISSUE THREAD

**Issue by sunnysideup** [titledk/silverstripe-calendar#34]
Fatal error: Call to undefined method MyEmptyEventPageDataExtension::stat() in /cms/code/model/SiteTree.php on line 408

**Comment by anselmdk** [comment]
It's working on `3.2`, haven't checked `3.3` yet, but I think I shouldn't be a big issue.
Have you tried installing https://github.com/titledk/silverstripe-calendar-demo locally? If you switch the composer version from `3.2` to `3.3` it should be fairly easy to point out what's the issue.

**Comment by sunnysideup** [comment]
yeah - sorry... I did not look very hard, but I definitely had a few issues with dev-master....
